app/checklist/views.py

```diff
@@ -88,3 +88,16 @@ def delete_default_checklist_item(id):
         flash('Error occurred. Please try again.', 'form-error')
         return redirect(url_for('checklist.index'))
     return redirect(url_for('checklist.index'))
+
+
+@checklist.route('/<int:id>/view', methods=['GET'])
+def view_all_checklist_items(id):
+    """View all checklist items, specific to each user"""
+    default_checklist_items = DefaultChecklistItem.query.all()
+    user_checklist_items = UserChecklistItem.query.filter_by(application_id=id)
+    return render_template(
+        'checklist/view_checklist_items.html',
+        default_checklist_items=default_checklist_items,
```

Comment:
Don't need to pass this in – a single user doesn't need to know what `default_checklist_items` is.

Reply:
Awesome!